ONBOARDING.md

```diff
@@ -0,0 +1,66 @@
+# Getting Started
+
+If you're here, that means you're interested in being a part of the volunteer effort. Welcome! We're glad to have you.
+
+The goal of this document is to give you a starting point for where to find the resources you need to get up to speed. This is by no means an exhaustive document, and I'm sure that we'll be missing some important items here, so if you're missing something, just ask and we will try to update this document.
+
+First thing's first - in order to keep track of everyone working on the project, please add yourself to the project roster: https://docs.google.com/spreadsheets/d/1jt4xrXWUnsh8DmtawNV4XGBj8RAdgn-d7ckgL2qEXPM/edit#gid=0
+
+Next - introduce yourself in the #intros channel on [Slack](https://join.slack.com/t/who-app/shared_invite/zt-cylowls6-Q7iQPvTAenkN_Yb9MLxVIw)!
+
+## Resources
+
+Where to find answers to your questions:
+
+* Github Issues: https://github.com/WorldHealthOrganization/app/issues
```

Comment:
```suggestion
* GitHub Issues: https://github.com/WorldHealthOrganization/app/issues
```

Reply:
fixed in https://github.com/WorldHealthOrganization/app/pull/104/commits/1c3f2b4d9fd88609d8a764d7e6c5b8880e374b6c